towhee/engine/operator_context.py

```diff
@@ -135,9 +134,8 @@ def num_finished_tasks(self) -> int:
         # self._finished_tasks' only monifier.
 
     def _write_outputs(self, task: Task):
```

Comment:
This function will be called in task-executor，so we must use the lock to make thread-safe